app/routes/index.tsx

```diff
@@ -1,11 +1,14 @@
 import { json, LoaderFunction, useLoaderData } from "remix";
-import App from "~/src/App";
-import fs from "fs"
-import { ParserData } from "parser/src/model";
+import { ParserData } from "~/parser/src/model";
 import appStyles from "~/styles/App.css"
 import varsStyles from "~/styles/vars.css"
 import indexStyles from "~/styles/index.css"
 import chartStyles from "~/styles/Chart.css"
+import { parse } from "~/parser/src/parse.server";
+import { Providers } from "~/src/components/Providers";
```

Comment:
Should we maybe start using export default for components to make the imports cleaner?

Reply:
Idk, I kinda like named exports, since it enforces you to import the exported value as the name it was defined as.